code/game/machinery/virtual_reality/vr_console.dm

```diff
@@ -1,227 +1,260 @@
+
+	return 1
+
+/obj/machinery/vr_sleeper/attackby(var/obj/item/I, var/mob/user)
+	add_fingerprint(user)
+
+	if(occupant && (istype(I, /obj/item/device/healthanalyzer) || istype(I, /obj/item/device/robotanalyzer)))
+		I.attack(occupant, user)
+		return
+
+	if(default_deconstruction_screwdriver(user, I))
+		return
+	else if(default_deconstruction_crowbar(user, I))
+		if(occupant && avatar)
+			avatar.exit_vr()
+			avatar = null
+			go_out()
+		return
+
+
+/obj/machinery/vr_sleeper/MouseDrop_T(var/mob/target, var/mob/user)
+	if(user.stat || user.lying || !Adjacent(user) || !target.Adjacent(user)|| !isliving(target))
+		return
+	go_in(target, user)
+
+
+
+/obj/machinery/sleeper/relaymove(var/mob/user)
+	..()
+	if(usr.incapacitated())
+		return
+	go_out()
+
+
+
+/obj/machinery/vr_sleeper/emp_act(var/severity)
+	if(stat & (BROKEN|NOPOWER))
+		..(severity)
+		return
+
+	if(occupant)
+		// This will eject the user from VR
+		// ### Fry the brain? Yes. Maybe.
+		if(prob(severity * 15) && occupant.species.has_organ[O_BRAIN] && occupant.internal_organs_by_name[O_BRAIN])
+			var/obj/item/organ/O = occupant.internal_organs_by_name[O_BRAIN]
+			O.take_damage(severity * 2)
+			visible_message("<span class='danger'>\The [src]'s internal lighting flashes rapidly, before the hatch swings open with a cloud of smoke.</span>")
+			smoke.set_up(n = severity, 0, src)
+			smoke.start("#202020")
+		go_out()
+
+	..(severity)
+
+/obj/machinery/vr_sleeper/verb/eject()
+	set src in view(1)
+	set category = "Object"
+	set name = "Eject VR Capsule"
+
+	if(usr.incapacitated())
+		return
+
+	var/forced
+
+	if(stat & (BROKEN|NOPOWER) || occupant && occupant.stat == DEAD)
+		forced = 1
+
+	go_out(forced)
+	add_fingerprint(usr)
+
+/obj/machinery/vr_sleeper/verb/climb_in()
+	set src in oview(1)
+	set category = "Object"
+	set name = "Enter VR Capsule"
+
+	if(usr.incapacitated())
+		return
+	go_in(usr, usr)
+	add_fingerprint(usr)
+
+/obj/machinery/vr_sleeper/relaymove(mob/user as mob)
+	if(user.incapacitated())
+		return 0 //maybe they should be able to get out with cuffs, but whatever
+	go_out()
+
+/obj/machinery/vr_sleeper/proc/go_in(var/mob/M, var/mob/user)
+	if(!M)
+		return
+	if(stat & (BROKEN|NOPOWER))
+		return
+	if(!ishuman(M))
+		to_chat(user, "<span class='warning'>\The [src] rejects [M] with a sharp beep.</span>")
+	if(occupant)
+		to_chat(user, "<span class='warning'>\The [src] is already occupied.</span>")
+		return
+
+	if(M == user)
+		visible_message("\The [user] starts climbing into \the [src].")
+	else
+		visible_message("\The [user] starts putting [M] into \the [src].")
+
+	if(do_after(user, 20))
+		if(occupant)
+			to_chat(user, "<span class='warning'>\The [src] is already occupied.</span>")
+			return
+		M.stop_pulling()
+		if(M.client)
+			M.client.perspective = EYE_PERSPECTIVE
+			M.client.eye = src
+		M.loc = src
+		update_use_power(2)
+		occupant = M
+
+		update_icon()
+
+		enter_vr()
+	return
+
+/obj/machinery/vr_sleeper/proc/go_out(var/forced = 1)
```

Comment:
Why does forced default to 1? In combination with the above point on the forced var above, this seems to me like it'll always pass 1/TRUE into this argument, which means the check on 187 will never occur?